packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { cloneElement, FC, HTMLAttributes, ReactElement, ReactNode } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme, styleMixins } from '../../themes';
+import { Tooltip, PopoverContent } from '../Tooltip/Tooltip';
+import { OnTagClick } from '../Tags/Tag';
+import { TagList } from '../Tags/TagList';
+
+interface ContainerProps extends HTMLAttributes<HTMLOrSVGElement> {
+  /** Customise the container html element for the card. Defaults to div */
+  tag?: keyof JSX.IntrinsicElements;
+  /** Content for the card's tooltip */
+  tooltip?: PopoverContent;
+}
+
+const CardContainer: FC<ContainerProps> = ({ tag = 'div', children, tooltip, ...props }) => {
+  const element = React.createElement(tag, props, children);
+  return tooltip ? (
+    <Tooltip placement="top" content={tooltip} theme="info">
+      {element}
+    </Tooltip>
+  ) : (
+    element
+  );
+};
+
+export interface CardInnerProps {
+  href?: string;
+}
+
+const CardInner: FC<CardInnerProps> = ({ children, href }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  return href ? (
+    <a className={styles.innerLink} href={href}>
+      {children}
+    </a>
+  ) : (
+    <>{children}</>
+  );
+};
+
+export interface Props extends ContainerProps {
+  /** Main heading for the Card **/
+  heading: ReactNode;
```

Comment:
```suggestion
  title: ReactNode;
```

Reply:
`title` is an [html attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) so that's why I decided not to override it here.

Reply:
Ah, you are right! Thanks